code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/templates/AEPPushTemplate.kt

```diff
@@ -28,10 +28,6 @@ import com.adobe.marketing.mobile.notificationbuilder.internal.util.Notification
  */
 internal sealed class AEPPushTemplate(data: NotificationData) {
 
-    // Message data payload for the push template
-    internal lateinit var messageData: MutableMap<String, String>
```

Comment:
I prefer retaining this but making the type `NotificationData` with additional method `putString`. See https://github.com/adobe/aepsdk-ui-android/pull/16/files#r1618052856

Reply:
After the discussion, 
Implemented the solution to hold the `NotificationData`property  for each template and pass that for construction of FallBack BasicPushNotificationTemplate.

Also discussed that we won't apply one of the random downloaded Carousel image for building BasicPushTemplate (During edge case when less than 3 images are downloaded for Carousel template)